syntax = "proto3";

package bakdata.corporatePerson.v1;

message CorporatePerson {
  uint32 id = 1;
  string firstname = 2;
  string lastname = 3;
  string city = 4;
  string birthdate = 5;
  string corporateName = 6;
  string corporateID = 7;
}